model_compression_toolkit/core/common/quantization/node_quantization_config.py

```diff
@@ -413,8 +411,6 @@ def __init__(self, qc: QuantizationConfig,
             if isinstance(attr, int):
                 # this is a positional attribute, so it needs to be handled separately.
                 # we assume that a positional attribute is quantized with the default configuration provided in the TPC.
-                if op_cfg.default_weight_attr_config.enable_weights_quantization:
```

Comment:
maybe change the log message to info and mention that this is an experimental feature